main.go

```diff
@@ -75,9 +78,6 @@ func main() {
 		helper.Logger.Info("Succeed to create message bus sender.")
 	}
 
-	// Read all *.so from plugins directory, and fill the variable allPlugins
-	allPluginMap := mods.InitialPlugins()
-
 	iam.InitializeIamClient(allPluginMap)
```

Comment:
Whether the `iam.InitializeIamClient(allPluginMap)` is a repeated work?

Reply:
emmm, probably not, it loads according to the name it chooses, and it is initialized only once